contributors/design-proposals/pod-priority-resourcequota.md

```diff
@@ -0,0 +1,81 @@
+# Priority in ResourceQuota
+
+@resouer
+
+Aug 2017
+  * [Objective](#objective)
+    * [Non-Goals](#non-goals)
+  * [Background](#background)
+  * [Overview](#overview)
+  * [Detailed Design](#detailed-design)
+    * [Expected behavior of ResourceQuota apply to pod](#resourcequota-apply-to-pod)
+    * [Expected behavior of ResourceQuota admission controller](#resourcequota-admission-controller)
+
+
+## Objective
+
+This feature is designed to make ResourceQuota become priority aware, three sub-tasks are included.
+
+* Add a field to ResourceQuota that specifies a priority class name.
```

Comment:
I agree `no PriorityClassName ResourceQuota applies to pod without PriorityClassName` is understandable for user.